doc/breadthFirstSearch/pgr_binaryBreadthFirstSearch.rst

```diff
@@ -14,16 +14,13 @@
   (`3.3 <https://docs.pgrouting.org/3.3/en/pgr_binaryBreadthFirstSearch.html>`__)
   `3.2 <https://docs.pgrouting.org/3.2/en/pgr_binaryBreadthFirstSearch.html>`__
 
-pgr_binaryBreadthFirstSearch - Experimental
+``pgr_binaryBreadthFirstSearch - Experimental``
 ===============================================================================
 
 ``pgr_binaryBreadthFirstSearch`` — Returns the shortest path(s) in a binary graph.
-Any graph whose edge-weights belongs to the set {0,X}, where 'X' is any non-negative real integer, is termed as a 'binary graph'.
+Any graph whose edge-weights belongs to the set {0,X}, where 'X' is any 
+non-negative real integer, is termed as a 'binary graph'.
 
-.. figure:: images/boost-inside.jpeg
```

Comment:
Do not remove the references to the boost functions

Reply:
Hi @cvvergara I am not sure where we need to use the boost graph inside reference. Actually, i referred the documentation of [pgr_edwardmoore](https://docs.pgrouting.org/latest/en/pgr_edwardMoore.html) and [pgr_chineasePostman](https://docs.pgrouting.org/latest/en/pgr_chinesePostman.html) . These docs are also not using the boost graph inside reference. So, I checked ther header files from include folder and I found that there is no direct implementation of these algorithm by boost. Similar case is for pgr_binaryBreadthFirstSearch

Reply:
Did you read the C++ code of the function?

Reply:
Yes mam, I have gone through the pgr_binaryBreadthFirstSearch.hpp file.
In other shortest path finding algorithm implementations, the private section of the main function class in header file consists of  call to the boost graph algorithm which is not the case for pgr_binaryBreadthFirstSearch.hpp It consists of local implementation.